docs/sink_connector.rst

```diff
@@ -0,0 +1,139 @@
+JDBC Sink Connector
+===================
+
+The JDBC sink connector allows you to export data from Kafka topics to any relational database with a JDBC driver.
+By using JDBC, this connector can support a wide variety of databases without requiring custom code for each one.
```

Comment:
Heh, might want to refine the end of this sentence as we technically have custom code for a bunch of different DBs. More like each database doesn't need its own dedicated connector implementation.